.buildkite/pipeline.ml.yml

```diff
@@ -153,9 +153,12 @@
     - conda create -n rllib_contrib python=3.8 -y
     - conda activate rllib_contrib
     - (cd rllib_contrib/a2c && pip install -r requirements.txt && pip install -e ".[development]")
+    - cp rllib/tests/run_regression_tests.py rllib_contrib/a2c/.
```

Comment:
Yes, we'll have to make sure this script, which will be "running along" with the evolving RLlib, will remain backward compatible wrt these "old" contrib tuned_examples files. This could potentially become an issue, for example if we decided to get rid of yaml support (and do everything with py config files). Let me think about this some more. Maybe a better alternative would be to copy this file from the actually installed (ray[rllib]) location ...

Reply:
I'm getting this script now via a wget from the 2.5.1 github location. So this will always be constant (backward compatible), no matter what we do with the master version of this script.